prow/plugins/assign/assign.go

```diff
@@ -49,7 +49,7 @@ func helpProvider(config *plugins.Configuration, enabledRepos []string) (*plugin
 		Usage:       "/[un]assign [[@]<username>...]",
 		Description: "Assigns an assignee to the PR",
 		Featured:    true,
-		WhoCanUse:   "Anyone can use the command, but the target user must be a member of the org that owns the repository.",
+		WhoCanUse:   "Anyone can use the command, but the target user must be a member of the org that owns the repository or should have commented on the issue or PR.",
```

Comment:
```suggestion
		WhoCanUse:   "Anyone can use the command, but the target user must be an org member, a repo collaborator, or should have previously commented on the issue or PR.",
```

Reply:
Thanks. Much better now.

Reply:
@nikhita Any comments? 

Maybe we should add `yourself` also, like follows: 
`but the target user must be an org member, a repo collaborator, yourself, or should have previously commented on the issue or PR.`